### DISCUSSION TITLE
Adjust taskbar icons

### DISCUSSION THREAD

**Discussion by gi-b716** [ppy/osu#28315] [Ideas] [answered]
In the game, the text in the OSU icon is at a distance from the border. But in the taskbar, the text looks like it tops the whole icon, which seems very strange. Suggest adjusting the taskbar icon to harmonize it with in-game.

---

In game:
![image](https://github.com/ppy/osu/assets/78394473/ad45fc8e-4646-49f5-b7e8-89820b52fbb8)\
On taskbar:
![image](https://github.com/ppy/osu/assets/78394473/099bcdf4-275e-43c9-8fac-a2b88560b566)

**Comment by SupDos** [comment]
This only happened after the dot was removed, and I'm assuming it was an oversight as peppy [mentioned](https://discord.com/channels/188630481301012481/1097318920991559880/1241722653140844654) it's only supposed to show like that at very small sizes

Comparison between v4b, v4 and stable

|   | v4b  | v4  | stable |
|:---:|:---:|:---:|:---:|
| 48px  | ![app-1](https://github.com/ppy/osu/assets/6813986/174d2134-088e-4f5d-a6f2-31a01113ff3c)  |  ![app-1](https://github.com/ppy/osu/assets/6813986/e35b1007-d1ce-46c2-bb19-2638fbc4dfa5) | ![stable-6](https://github.com/ppy/osu/assets/6813986/cec77640-b9d9-4e37-aae1-137ad46fce1c)  |
| 32px  | ![app-2](https://github.com/ppy/osu/assets/6813986/732c3a95-244f-4689-8495-256c5b767fa6)  | ![app-2](https://github.com/ppy/osu/assets/6813986/bd4374cf-62f3-4adc-8b1b-c1e963f5e6c6)  | ![stable-7](https://github.com/ppy/osu/assets/6813986/da49b0df-15e3-4b91-9e03-03dbdfda9ff5) |
| 16px | ![app-3](https://github.com/ppy/osu/assets/6813986/9990da9f-325f-4da2-8999-aed7d0347406) |![app-3](https://github.com/ppy/osu/assets/6813986/24889ad3-49d8-4c1d-a330-95321ba6199c)  | ![stable-2](https://github.com/ppy/osu/assets/6813986/53f128e9-2ea8-4f12-acfe-75517f3c0536) |

I think the new spacing is only supposed to be used for the 16px icon, to match stable's tiny text version.

**Reply by peppy** [reply]
yeah unfortunately i took @arflyte 's icons at face value. 48px is way too large to be using this ugly version.

**Reply by peppy** [reply]
@arflyte your low res version doesn't work at 16px. I'm going to use the stable one.

**Reply by SupDos** [reply]
I'd add that 32px is also probably too large for it - I believe that's the size the Windows taskbar uses for the icon, and is what the OP of the discussion was asking about originally.

**Reply by peppy** [reply]
correct - it should only be used in 16px sizes like the older icon.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/28327

**Reply by arflyte** [reply]
I believe we discussed this in the office before that we are using stable's for 16px coz it's pretty much pixelart at that size.

**Reply by peppy** [reply]
Okay sure. Then just scrap the 48px version because it looks wrong.
